client/Gruntfile.js

```diff
@@ -57,9 +57,8 @@ module.exports = function (grunt) {
         'clean',
         'style',
         'template:mock',
-        'connect:mock',
         'ngtemplates',
-        'watch'
```

Comment:
It was never used before because never executed, so I assumed that nobody needed it.
And for development, it makes sense to watch files, but not for e2e tests from my point of view.